docs/toast-uncontrolled.md

```diff
@@ -0,0 +1,69 @@
+---
+id: toastUncontrolled
+title: Uncontrolled Toast
+---
+
+## Static
+
+For the static method to work the root component must be wrapped by `<Provider />`, since it needs rendered `<ToastContainer />` to work properly.
+
+- `showToast(params)` - show a Toast and then dismiss it after the specified duration has passed
```

Comment:
Let's move this whole thing to the bottom of the page, not just the `hideToast` alone.

Reply:
And also don't forget to change both `showToast` and `hideToast` to use the same format as the params list, not the current format.